arches/app/etl_modules/branch_excel_importer.py

```diff
@@ -24,15 +24,39 @@
 
 
 class BranchExcelImporter(BaseImportModule):
-    def __init__(self, request=None, loadid=None, temp_dir=None):
-        self.request = request if request else None
-        self.userid = request.user.id if request else None
+    def __init__(self, request=None, loadid=None, temp_dir=None, params=None):
```

Comment:
Certainly not necessary for this PR, but we should probably stop passing request objects to the importer, but rather the data needed from the request object.

Reply:
Handling the request separately from the module so that the module can be used without the request... something like that? Yeah, I was thinking the same thing.